packages/react-search-ui/src/containers/Facet.tsx

```diff
@@ -93,9 +93,11 @@ export class FacetContainer extends Component<
 
     if (searchTerm.trim()) {
       facetValues = facetValues.filter((option) =>
-        accentFold(option.value)
-          .toLowerCase()
-          .includes(accentFold(searchTerm).toLowerCase())
+        typeof option.value === "string"
+          ? accentFold(option.value)
```

Comment:
Found that `option.value` is not always string. Especially, when using a `<Facet isFilterable={true} />` on a field mapped to `NUMBER` results